.github/workflows/.trivyignore

```diff
@@ -1,4 +0,0 @@
-# Date: Feb 12, 2024
```

Comment:
Can you add the following condition to the "Trivy - Stop on severe vulnerabilities" action:

`if github.event_name == 'pull_request' `

That way trivy will only break pipeline on PRs and not merges to main

Reply:
:+1: